Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'bitters', '~> 1.2.0'
 gem 'bixby'
 gem 'blacklight', '~> 6.10'
 gem 'blacklight-gallery', '~> 0.4'
-gem 'blacklight_range_limit', '~> 6.1'
+gem 'blacklight_range_limit', git: 'https://github.com/projectblacklight/blacklight_range_limit', branch: 'rails-5.1'
```

Comment:
Can you please add a comment explaining why this branch is required and what conditions should be met to go back to a release?